CHANGELOG.md

```diff
@@ -71,6 +71,7 @@
 
 ### Internal
 * Uses Realm Core v12.9.0
+* Added tracking of child handles for objects/results/lists obtained from an unowned Realm. This ensures that all children are invalidate as soon as the parent Realm gets released at the end of the callback. (Issue [#527](https://github.com/realm/realm-dart/issues/527))
```

Comment:
```suggestion
* Added tracking of child handles for objects/results/lists obtained from an unowned Realm. This ensures that all children are invalidated as soon as the parent Realm gets released at the end of the callback. (Issue [#527](https://github.com/realm/realm-dart/issues/527))
```

Reply:
Need to install a proofreader in VS Code 😅